docs/source/contributing.rst

```diff
@@ -0,0 +1,121 @@
+Contributing
```

Comment:
This is a great addition, can you also include the quality checks that we perform in this? For example, black and isort?